src/Service.GraphQLBuilder/Mutations/CreateMutationBuilder.cs

```diff
@@ -539,21 +557,22 @@ public static NameNode GenerateInputTypeName(string typeName)
 
             createMutationNodes.Add(createOneNode);
 
-            if (IsRelationalDb(databaseType) && DoesRelationalDBSupportMultipleCreate(databaseType))
+            // Multiple create node is created in the schema only when multiple create operation is enabled.
+            if (isMultipleCreateOperationEnabled && IsRelationalDb(databaseType) && DoesRelationalDBSupportMultipleCreate(databaseType))
```

Comment:
Rather than using independent conditions: `isMultipleCreateOperationEnabled &&  DoesRelationalDBSupportMultipleCreate(databaseType)`, can we create one method which takes care of everything? Here we have two variables:

1. Database type
2. Feature-flag: set to true/false.

Obviously, I don't anticipate multiple-mutations to be supported for multiple database scenario soon. So both the above two things are known to RuntimeConfig. So maybe that might be a good place to have this method?

On a side note, I see that the RuntimeConfig.Entities are also passed on throughout all these method calls. So maybe if we can just pass the runtime config through these methods, we can take care of both the things.

Reply:
Ack. Evaluating this.

Reply:
I've introduced a method called `IsMultipleCreateOperationSupportedAndEnabled()` in `RuntimeConfig` class that essentially combines the checks: a) Whether multiple create is enabled through the config file and b) Whether the database type supports multiple create operation. 

I'm still continuing to pass just the `RuntimeEntities` and `IsMultipleCreateOperationSupportedAndEnabled` parameters instead of the complete `Runtime` object as a parameter - this is because, only these 2 fields from the `RuntimeConfig` object are required. The other fields are not necessary, and hence we don't necessarily need the entire `RuntimeConfig` object.

Reply:
Changing `CreateMutationBuilder.Build()`  to accept `RuntimeObject` is purely a refactor change. IMO, we don't need the complete `RuntimeConfig` object, as only `RuntimeEntities` and `IsMultipleCreateOperationEnabled` fields are relevant for schema generation; hence, we can continue to pass them separately. 

I've created the https://github.com/Azure/data-api-builder/issues/2132 for further evaluation/discussion on this.

Since, this is purely a refactor change, this can be picked up in a follow-up PR (if we make the decision to go ahead with the refactor). This need not hold this PR from merging. 

Resolving this thread